lib/ask/runtime/session.ex

```diff
@@ -121,7 +121,7 @@ defmodule Ask.Runtime.Session do
   end
 
   defp url(respondent_id) do
-    shorten("#{mobile_base_url()}/mobile_survey/#{respondent_id}?token=#{Respondent.token(respondent_id)}")
+    shorten("#{mobile_base_url()}/mobile/#{respondent_id}?token=#{Respondent.token(respondent_id)}")
```

Comment:
Have you tried building the URL using the helper?

```elixir
shorten(Ask.Router.Helpers.mobile_survey_url(Ask.Endpoint, :index, respondent_id, token: Respondent.token(respondent_id))})`
```

I'm not sure if the host will be correct, thought 🤔

Reply:
yeah, I thought about it but wasn't sure if it was a good practice using the Helper for this and didn't know how to set a custom url with the helper as in `mobile_base_url()`
I could replace it though with
```elixir
shorten("#{mobile_base_url()}#{mobile_survey_path(Ask.Endpoint, :index, respondent_id, token: Respondent.token(respondent_id))})
```